src/FirstSteps/RPG/AdventuresRegistry.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FirstSteps.RPG.Adventures;
+using Spectre.Console;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirstSteps.RPG
+{
+    public class AdventuresRegistry 
+    {
+        private static List<IAdventure> _adventures = new List<IAdventure>()
+        {
+          new Dungeons(),
+          new Forest(),
+          new Hell(),
+          new Mine(),
+          new TreasureChest(),
+          new Village(),
+          new Inventory()
+        };
+        
+        public void GetMenu()
```

Comment:
It should not be here. Let's focus on the issue description. You have to create `a public method called GetCommands that returns a list of all available adventure commands`. So it should look like this `public List<string> GetCommands()`. You can get the commands from `_adventures`  and as I wrote previously the [Select](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=net-7.0#system-linq-enumerable-select-2(system-collections-generic-ienumerable((-0))-system-func((-0-system-int32-1)))) method can be useful.